peps/pep-0728.rst

```diff
@@ -647,12 +644,26 @@ For example::
     int_mapping: Mapping[str, int] = extra_int  # Not OK. 'int | str' is not assignable with 'int'
     int_str_mapping: Mapping[str, int | str] = extra_int  # OK
 
-Type checkers should be able to infer the precise return types of ``values()``
-and ``items()`` on such TypedDict types::
+Type checkers should be able to infer the precise signatures of ``values()``,
```

Comment:
Does "should be able to" imply that conformant type checkers _must_ implement this, or is it saying that type checkers may implement this? The PEP should make this clear by using the standardized terms "should" or "may". The phrase "should be able to" leaves it in a gray area.